LibCV/PrePARE/PrePARE.py

```diff
@@ -634,6 +633,15 @@ def ControlVocab(self, ncfile, variable=None, print_all=True):
                 print("=====================================================================================")
                 print(BCOLORS.ENDC)
                 self.errors += 1
+        # Check if cell_measures is defined in the file but not in the table
```

Comment:
@taylor13 I added this new section to check if cell_measures got defined in the file but not in the table.